### PR TITLE
SRCH-722-Fix twitter data rspec errors

### PR DESCRIPTION
**Pull request by peggles2**
SRCH-722 - Fix rspec errors for the twitter_data_spec

### CODE REVIEW

File: spec/lib/importers/twitter_data_spec.rb [hunk 1]
```diff
@@ -58,24 +58,24 @@
...
+      it 'should append the imported lists to the profile' do
         allow(profile).to receive_message_chain(:twitter_lists, :exists?).with(id: ar_list.id).and_return(false)
-        expect(profile.twitter_lists).to receive(:<<).with(ar_list)
+        expect(profile.twitter_lists).to receive(:push).with(ar_list)
         TwitterData.import_twitter_profile_lists(profile)
       end
     end
 
     context 'when the profile twitter lists include the imported list' do
       xit 'should not append the imported lists to the profile' do
```
Comment (MothOnMars) [inline spec/lib/importers/twitter_data_spec.rb]: One more test to re-enable

**Comment by peggles2** [comment]
Thanks for the review Martha. But I did not request review yet. Noticed rspec was barfing a bunch of errors so was trying to figure that out first. :)

**Comment by MothOnMars** [comment]
Sorry, @peggles2 - I jumped the gun!

**Comment by peggles2** [comment]
No worries! all good :)

**Comment by peggles2** [comment]
Okay ready for review

**Pull request merged**